src/main/java/org/broadinstitute/hellbender/engine/dataflow/datasources/RefAPISource.java

```diff
@@ -0,0 +1,122 @@
+package org.broadinstitute.hellbender.engine.dataflow.datasources;
+
+import com.google.api.services.genomics.Genomics;
+import com.google.api.services.genomics.model.ListBasesResponse;
+import com.google.api.services.genomics.model.Reference;
+import com.google.api.services.genomics.model.ReferenceSet;
+import com.google.cloud.dataflow.sdk.options.PipelineOptions;
+import com.google.cloud.genomics.dataflow.utils.GCSOptions;
+import com.google.cloud.genomics.dataflow.utils.GenomicsOptions;
+import com.google.cloud.genomics.utils.GenomicsFactory;
+import htsjdk.samtools.util.Locatable;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.reference.ReferenceBases;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.security.GeneralSecurityException;
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * RefAPISource makes calls to the Google Genomics API to get ReferenceBases. This is designed to used with workers.
+ * The service is created on each worker. We've benchmarked this and found that class instantiation is much cheaper
```

Comment:
I have no idea what the benchmark comment is trying to tell me. Which API call? Which class instantiation? Why are those two things comparable? Ideally I'd like for the benchmark to be included in the code, but here this may not be necessary.